Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -170,6 +170,8 @@ Supporting infrastructure and systems MAY include:
 - resources to support redundancy
 - resources to support failover
 
+If your boundary includes on-premise and/or cloud data center operations, E should take into account the efficiency of the data center, including cooling and other energy consumption necessary to operate a data center. The data center's energy efficiency is usually available as a PUE(Power Usage Effectiveness) value.
```

Comment:
@GadhuNTTDATA I think it might be useful to add a reference (canonical if available and possible) on the PUE so that people know what exactly we're referring to.